handler_templates.ts

```diff
@@ -384,6 +392,12 @@ export function untransformBody(body: any, schema: Schema | undefined): any {
   return body;
 }
 
+export function untransformKeys(keys: string[], schema: Schema | undefined): string[] {
```

Comment:
Unlike the other methods (mapKeys vs unmapKeys, transformBody vs untransformBody), it's unclear by name what this method is used for and when it should be used. Can you add some documentation to this method?

Reply:
Added comment plus some tests which I realized I hadn't committed.